1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,26 @@
 // Example 1
 let a;
 console.log(a);
-
+/* a is not defined */
 
 // Example 2
 function sayHello() {
     let message = "Hello";
 }
-
+/* there is no return */
 let hello = sayHello();
 console.log(hello);
-
+/* hello is a variable not a function */
```

Comment:
In javascript, functions can be assigned to variables e.g. 
```
const hello = sayHello
hello()
```
So the reason that it is undefined is because you assign `hello` to the result of calling `sayHello()`; as `sayHello()` doesn't return anything, `hello` is `undefined`.